parser_library/src/processing/preprocessors/preprocessor_utils.h

```diff
@@ -20,6 +21,9 @@
 #include <regex>
 #include <vector>
 
+#include "semantics/range_provider.h"
```

Comment:
does not seem to be actually needed, move to cpp?

Reply:
It's now needed (again) in order to be able to just reused `range_provider` created in a preceding call...